assets/js/ajax-filters.js

```diff
@@ -1,288 +1,458 @@
+				data: data,
+				success: function( result ) {
+					if ( result ) {
+						try {
+							result.data = data;
+
+							job_manager_handle_result( $target, result, append );
+
+							$results.removeClass( 'loading' );
+							$target.triggerHandler( 'updated_results', result );
+
+							job_manager_save_results( $target, result );
+						} catch ( err ) {
+							if ( window.console ) {
+								window.console.log( err );
+							}
+						}
+					}
+				},
+				error: function( jqXHR, textStatus, error ) {
+					if ( window.console && 'abort' !== textStatus ) {
+						window.console.log( textStatus + ': ' + error );
+					}
+				},
+				statusCode: {
+					404: function() {
+						if ( window.console ) {
+							window.console.log(
+								'Error 404: Ajax Endpoint cannot be reached. Go to Settings > Permalinks and save to resolve.'
+							);
+						}
+					},
+				},
+			} );
+		} );
+
+	$(
+		'#search_keywords, #search_location, .job_types :input, #search_categories, .job-manager-filter'
+	)
+		.change( function() {
+			var $target = $( this ).closest( 'div.job_listings' );
+			$target.triggerHandler( 'update_results', [ 1, false ] );
+			job_manager_store_state( $target );
+		} )
+		.on( 'keyup', function( e ) {
+			if ( e.which === 13 ) {
+				$( this ).trigger( 'change' );
+			}
+		} );
+
+	$( '.job_filters' )
+		.on( 'click', '.reset', function() {
+			var $target = $( this ).closest( 'div.job_listings' );
+			var $form = $( this ).closest( 'form' );
+
+			$form
+				.find(
+					':input[name="search_keywords"], :input[name="search_location"], .job-manager-filter'
+				)
+				.not( ':input[type="hidden"]' )
+				.val( '' )
+				.trigger( 'change.select2' );
+			$form
+				.find( ':input[name^="search_categories"]' )
+				.not( ':input[type="hidden"]' )
+				.val( '' )
+				.trigger( 'change.select2' );
+			$( ':input[name="filter_job_type[]"]', $form )
+				.not( ':input[type="hidden"]' )
+				.attr( 'checked', 'checked' );
+
+			$target.triggerHandler( 'reset' );
+			$target.triggerHandler( 'update_results', [ 1, false ] );
+			job_manager_store_state( $target );
+
+			return false;
+		} )
+		.on( 'submit', function() {
+			return false;
+		} );
+
+	$( document.body ).on( 'click', '.load_more_jobs', function() {
+		var $target = $( this ).closest( 'div.job_listings' );
+		var page = parseInt( $( this ).data( 'page' ) || 1, 10 );
+
+		$( this ).addClass( 'loading' );
+
+		page = page + 1;
+		$( this ).data( 'page', page );
+
+		$target.triggerHandler( 'update_results', [ page, true ] );
 		return false;
 	} );
 
 	if ( $.isFunction( $.fn.select2 ) && typeof job_manager_select2_args !== 'undefined' ) {
 		var select2_args = job_manager_select2_args;
-		select2_args['allowClear']              = true;
-		select2_args['minimumResultsForSearch'] = 10;
+		select2_args[ 'allowClear' ] = true;
+		select2_args[ 'minimum$resultsForSearch' ] = 10;
```

Comment:
Is the change to `minimum$resultsForSearch` correct?

Reply:
Fixed in e82e1543946ba67dfa5bab01ad1cc9f7ebc7e57b 😄